addon/components/variable-plugin/autofilled/insert.hbs

```diff
@@ -0,0 +1,32 @@
+{{! @glint-nocheck: not typesafe yet }}
+<AuFormRow>
+  <VariablePlugin::Utils::LabelInput
+    @label={{this.label}}
+    @updateLabel={{this.updateLabel}}
+  />
+</AuFormRow>
+<AuFormRow>
+  <AuLabel for='autofill_key'>
+    {{t 'variable-plugin.autofill.autofillKey'}}
+  </AuLabel>
+  <AuNativeInput
```

Comment:
I think it would make more sense to me as a powerselect, with the possible values taken from configuration. otherwise there is too much margin for error.

Reply:
Discussed with Elena but will be happy to discuss with you in a standup or we can set up a meeting, the idea is that the plugin is as flexible as possible and the template app have no idea which final values will be implemented in the filler app, this of course has inconvenients as it has margin for error

Reply:
Hmm, that makes sense but it still seems error-prone since we know at least some common ones that are likely to exist. Happy to just go with a text input though.